1_html-css/styles.css

```diff
@@ -0,0 +1,135 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  background-color: #f7f7f7;
+}
+html {
+  scroll-behavior: smooth;
+}
+
+.header-wrapper {
+  background-color: #00587a;
+  color: #fff;
+  padding: 20px;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  cursor: pointer;
+}
+
+.nav-links ul {
+  list-style: none;
+  display: flex;
+}
+
+.nav-links li {
+  margin-right: 20px;
+}
+
+.introduction h1 {
+  font-size: 36px;
+}
+
+.introduction p {
+  font-size: 18px;
+}
+.introduction {
+  text-align: center;
+  margin-top: 20px;
+  animation: fadeScale 2s ease-in-out infinite;
+}
+
+@keyframes fadeScale {
+  0%,
+  100% {
+    opacity: 0.5;
+    transform: scale(1);
+  }
+  50% {
+    opacity: 1;
+    transform: scale(1.1);
+  }
+}
+
+#about-me img {
+  width: 15%;
+  margin-bottom: 15px;
+}
+
+section {
+  padding: 40px;
+  background-color: #fff;
+  margin-bottom: 30px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+section h2 {
+  font-size: 28px;
+  margin-bottom: 20px;
+}
+
+section h3 {
+  font-size: 18px;
+  line-height: 1.6;
+  margin-bottom: 20px;
+}
+section p,
+.fa-github {
+  margin-bottom: 10px;
+}
+
+ul {
+  list-style: none;
+}
+
+ul li {
+  margin-bottom: 10px;
+}
+
+a {
+  color: #0072b1;
+  text-decoration: none;
+}
+.social-icons {
+  margin-top: 20px;
+  font-size: 28px;
+  gap: 20px;
+}
+
+/* Footer styles */
+footer {
+  background-color: #333;
+  color: #fff;
+  text-align: center;
+  padding: 20px;
+}
+
+/* Responsive styles */
+@media screen and (max-width: 768px) {
+  .navbar {
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .nav-links {
+    display: none;
```

Comment:
This hides the nav links in the mobile view, so maybe not so much point to style them further in they are hidden and maybe hiding is not what you intended 🙏🏽 

This could be good to look into and try to imitate!
https://www.w3schools.com/howto/howto_js_topnav_responsive.asp

Reply:
I actually meant to hide the nav links when the screen gets smaller but in the new version I remove that css property and let them be in left corner :)